docs/intro.rst

```diff
@@ -99,6 +99,8 @@ A quick example to showcase how events work:
 
 .. code-block:: python3
 
+    # This example requires the 'message_content' intent.
+
```

Comment:
```suggestion
```

`intents.message_content = True` already indicates this so this isn't really needed and to stay consistent with other examples across the documentation or GitHub/PyPi readme's.

Reply:
I have looked at other examples like the one in [Quickstart](https://github.com/Rapptz/discord.py/blob/master/docs/quickstart.rst#a-minimal-bot) and noticed that the same is being used there. So I guess it is actually telling the user to toggle it in the developer portal.